Assets/_BForBoss/Scripts/UserInterface/SettingsViewBehaviour.cs

```diff
@@ -0,0 +1,41 @@
+using PerigonGames;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.UI;
+
+namespace BForBoss
+{
+    public class SettingsViewBehaviour : MonoBehaviour
+    {
+        [SerializeField] private Button _backButton = null;
+        
+        private InputSettingsViewBehaviour InputSettingsView => GetComponentInChildren<InputSettingsViewBehaviour>();
+        private InputUsernameViewBehaviour InputUsernameView => GetComponentInChildren<InputUsernameViewBehaviour>();
```

Comment:
if these are only being used in `Initialize()`, then is there any point in creating these getters?

Instead of just doing `GetComponentInChildren<InputSettingsViewBehaviour>.Initialize(new InputSettingsViewModel(inputSettings));`

Reply:
it's just readability